package.json

```diff
@@ -12,12 +12,13 @@
     "@types/node": "20.5.3",
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
-    "firebase": "^10.4.0",
+    "firebase": "^10.5.0",
     "js-cookie": "^3.0.5",
     "mixpanel-browser": "^2.47.0",
     "next": "^13.4.19",
     "next-pwa": "^5.6.0",
     "pino": "^8.16.1",
+    "playwright": "^1.39.0",
```

Comment:
@Ntwalii where is this dependency used- else delete

Reply:
The dependency has been deleted